src/App.js

```diff
@@ -7,12 +7,15 @@ import "./styles/styles.css"
 
 export default function App() {
   const [contacts, setContacts] = useState([])
+  const [loading, setLoading] = useState(false)
   
   useEffect(() => {
+    setLoading(true)
```

Comment:
I like this set loading boolean. I've done the same thing and it's a really nice way to create this functionality!